src/genie/libs/parser/iosxe/tests/ShowWlanIdClientStats/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,206 @@
+            "iplearn": 25,
+            "l2auth": 882,
+            "l3auth": 0,
+            "l3vlan_override": 0,
+            "macauth": 0,
+            "mobility": 0,
+            "sm_session_creation": 0,
+            "static_ip": 0,
+        },
+        "datapath_plumb": 0,
+        "db_error": 0,
+        "deauth_request": 0,
+        "delete_from_ap": 0,
+        "dot11": {
+            "ap_insufficient_bandwidth": 0,
+            "assocation_fail": 0,
+            "auth_fail": 0,
+            "denied_data_rates": 0,
+            "denied_unspecified": 0,
+            "fail": 0,
+            "groupcipher_validation_failed": 0,
+            "ie_validation_failed": 0,
+            "invalid_akm": 0,
+            "invalid_ft_ie": 0,
+            "invalid_mdie": 0,
+            "invalid_pairwise_cipher": 0,
+            "invalid_pkmid": 74,
+            "invalid_qos_parameter": 0,
+            "invalid_rsnie_capabilities": 0,
+            "max_sta": 0,
+            "qos_policy": 0,
+            "sae_auth_fail": 0,
+            "sae_invalid": 0,
+            "unsupported_client": 0,
+            "unsupported_rsn_version": 0,
+        },
+        "dot11v": {
+            "association_fail": 0,
+            "preauth_fail": 0,
+            "rssi_low_threshold": 0,
+            "smart_roam_fail": 0,
+            "timer_timeout": 0,
+        },
+        "dot11w_max_sa": 0,
+        "dot11x_auth_timeout": 0,
+        "dot11x_credential_fail": 0,
+        "dot1x_no_config": 0,
+        "eap": {
+            "bad_keyframe": 0,
+            "invalid_key_type": 0,
+            "key_ack_unexpected": 0,
+            "key_error_unexpected": 0,
+            "key_install_unexpected": 0,
+            "key_mic_expected": 0,
+            "key_secure_unexected": 0,
+            "key_version_mismatch": 0,
+            "mic_validation_failed": 7,
+        },
+        "eogre": {
+            "aaa_override_error": 0,
+            "all_gateways_down": 0,
+            "domain_shut": 0,
+            "empty_domain": 0,
+            "flex_no_gateway": 0,
+            "generic_join_fail": 0,
+            "ha_reconcile": 0,
+            "invalid_domain": 0,
+            "invalid_gateway": 0,
+            "invalid_vlan": 0,
+            "mobility_handoff_error": 0,
+            "onboarding_error": 0,
+            "reset": 0,
+            "rule_error": 0,
+        },
+        "fqdn": {
+            "filter_missing": 0,
+            "invalid_group_id": 0,
+            "wrong_postauth_filter": 0,
+            "wrong_preauth_filter": 0,
+        },
+        "ha_recovery": 0,
+        "iapp_disassociate_wired": 0,
+        "idle_timeout": 0,
+        "internal_error": 0,
+        "ip_theft": 0,
+        "ip_update_timeout": 0,
+        "ip_zone_change": 0,
+        "l3_auth_fail": 0,
+        "mac": {"auth_fail": 0, "ip_theft": 0, "theft": 0},
+        "manually_excluded": 0,
+        "mobility": {"fail": 0, "peer_delete": 0, "tunnel_down": 0, "wlan_down": 0},
```

Comment:
maybe `fail` need to be outside of `mobility`?
```
   Due to mobility failure                                         : 0
```
```
"due_to_mobility_failure": 0,
"mobility": {
    (snip)

Reply:
Updated!